src/prime_bullet/camera.py

```diff
@@ -89,8 +89,11 @@ def render(self):
         rgb   = np.reshape(rgba, (ih, iw, 4))[:, :, :3]
         depth = np.reshape(d, (ih, iw))
         self.__current_rgb = rgb
-        self.__current_d   = depth 
```

Comment:
This is not ok, as user would expect true depth to be returned and not the opengl depth